packages/reactivity/src/effect.ts

```diff
@@ -122,62 +101,33 @@ export class ReactiveEffect<T = any> {
     if (!this.active) {
       return this.fn()
     }
-    let parent: ReactiveEffect | undefined = activeEffect
     let lastShouldTrack = shouldTrack
-    while (parent) {
-      if (parent === this) {
-        return
-      }
-      parent = parent.parent
-    }
+    let lastEffect = activeEffect
     try {
-      this.parent = activeEffect
-      activeEffect = this
       shouldTrack = true
-
-      trackOpBit = 1 << ++effectTrackDepth
-
+      activeEffect = this
+      this._runnings++
       cleanupEffect(this)
       return this.fn()
     } finally {
-      if (effectTrackDepth <= maxMarkerBits) {
-        finalizeDepMarkers(this)
```

Comment:
I see that you no longer clean up dependencies after running the effect. This indeed saves performance.

However, consider this situation:
```javascript
const a = ref(true)
const b = computed(() => ...)
const c = computed(() => ...)
const d = computed(() => if (a.value) b.value else c.value)

console.log(d.value)
a.value = false
console.log(d.value)
triggerRef(b)
```
As far as I can see in your code, the final triggerRef will also trigger `d`, while it shouldn't.
This is because the dependencies are never cleaned up. I first thought that the `trackId` would cover this, but it is not incremented on each run and only serves to detect cleanup.

Am I overlooking something?